flexmeasures/api/common/schemas/assets.py

```diff
@@ -10,6 +11,23 @@ class AssetAPIQuerySchema(PaginationSchema):
         validate=validate.OneOf(["id", "name", "owner"]),
     )
     account = AccountIdField(data_key="account_id", load_default=None)
+    root_asset = AssetIdField(
+        data_key="root",
+        load_default=None,
+        metadata=dict(
+            description="Select all descendants of a given root asset (including the root itself). Leave out to select top-level assets.",
+            example=482,
+        ),
+    )
+    max_depth = fields.Int(
+        data_key="depth",
+        validate=validate.Range(min=0),
+        load_default=None,
+        metadata=dict(
+            description="Maximum number of levels of descendant assets to include. Set to 0 to include root assets only.",
```

Comment:
We could mention what the default behavior is. If we leave this out, then the whole tree is returned, I believe.